app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -310,11 +310,6 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
-    var shouldUseAutoSize by booleanPreference(
```

Comment:
I am wondering if we can turn this into a getter (might not have to be in Settings or use the same name) and return the boolean value of `fontSizeFactor == 1F`. The way I think about this is that `fontSizeFactor == 1F` could be abstracted into a getter and with the right naming could help others reason what the setting means when looking at the other classes and not having the context of font size resizing.

Reply:
Good call out! Changed to reflect that and make it more usable